package-testing/android-relay/app/src/main/java/cloud/eppo/android/sdkrelay/EppoSdkRelay.kt

```diff
@@ -0,0 +1,5 @@
+package cloud.eppo.android.sdkrelay
+
+import android.app.Application
+
+class EppoSdkRelay : Application() {}
```

Comment:
nit: Its good idea to suffix with application here for clarity - `EppoSdkRelayApplication`

Reply:
🙏